docs/GettingStarted.md

```diff
@@ -1093,9 +1093,31 @@ end
 
 `options` are the following keyword arguments:
 
-| Key       | Env Var                         | Type   | Description                                  | Default |
-| --------- | ------------------------------- | ------ | -------------------------------------------- | ------- |
-| `enabled` | `DD_TRACE_KAFKA_ENABLED` | `Bool` | Whether the integration should create spans. | `true` |
+| Key       | Env Var                  | Type   | Description                                  | Default |
+| --------- | ------------------------ | ------ | -------------------------------------------- | ------- |
+| `enabled` | `DD_TRACE_KAFKA_ENABLED` | `Bool` | Whether the integration should create spans. | `true`  |
+
+### Karafka
```

Comment:
Can you remove the unrelated changes in the `GettingStarted.md`, and leave the newly introduced Karafka changes only?

Reply:
ahh, my auto correction setting, thanks @marcotc, will discard them